tensorflow/python/keras/BUILD

```diff
@@ -119,6 +119,7 @@ py_library(
     deps = [
         ":backend",
         "//tensorflow/python/keras/utils:engine_utils",
+        "//tensorflow/python/keras/layers:advanced_activations",
```

Comment:
Ah, there's a small issue here. Can you swap lines 121 and 122? (deps should appear in alphabetical order)

Reply:
ah good to know, sorry about that. Fixed! :)